apps/studio/src/server/modules/searchsg/searchsg.service.ts

```diff
@@ -0,0 +1,125 @@
+import wretch from "wretch"
+
+import { env } from "~/env.mjs"
+
+const SEARCHSG_BASE_URL = "https://api.services.search.gov.sg/admin"
```

Comment:
issue: we can't test this in non-prod environment?

Reply:
lemme cross check with them!